static/styles/right_sidebar.css

```diff
@@ -113,14 +113,7 @@
     width: calc(100% - 24px);
 
     .status_emoji {
-        height: 16px;
-        width: 16px;
-        /* We are setting minimum width here because when the user's name is very long,
-        emoji's width decreases and causes it to break. */
-        min-width: 16px;
-        margin-left: 4px;
-        margin-right: 4px;
-        top: 8px;
+        top: 9px;
```

Comment:
Why the 8px => 9px change?

Reply:
I'm not exactly sure _why_ it has to be different here, but this is pixel perfect (ie the same as other places) whereas 8px would be 1px off.

Reply:
OK.  Also going to move this to `app_components.css`; we put reusable CSS there, not in `zulip.css` which we're trying to slowly kill.

Reply:
Hmm, maybe worth doing later, since we want to move `emoji` at the same time.